Source/WebCore/animation/ViewTimeline.h

```diff
@@ -55,6 +55,9 @@ class ViewTimeline final : public ScrollTimeline {
     Element* subject() const { return m_subject.get(); }
     const CSSNumericValue& startOffset() const;
     const CSSNumericValue& endOffset() const;
+
+    float startInsetValue() const { return m_insets.start ? m_insets.start->value() : 0; }
+    float endInsetValue() const { return m_insets.end ? m_insets.end->value() : startInsetValue(); }
```

Comment:
These should be `private` since the only call is within `ViewTimeline`. Even better, you could remove those methods since there is a single call site. Finally, you can use `value_or()` here which will remove the need for the ternary.

Reply:
Fixed.